Gui/opensim/view/src/org/opensim/threejs/ModelVisualizationJson.java

```diff
@@ -1346,11 +1344,15 @@ public void propagateGeometryPathCommandsToPathPoints(Force force, AbstractPrope
         }
         else {
             JSONObject pathpointCommand = (JSONObject) lastCommand.clone();
-            
-            GeometryPath gPath = GeometryPath.safeDownCast(force.getPropertyByName("GeometryPath").getValueAsObject());
-            
-            pathpointCommand.put("objectUuid", getFirstPathPointUUID4GeometryPath(gPath).toString());
-            commands.add(pathpointCommand);
+            if (force.hasGeometryPath()){
+                GeometryPath gPath = GeometryPath.safeDownCast(force.getPropertyByName("GeometryPath").getValueAsObject());
```

Comment:
It is still possible for there to be an exception here. There is no guarantee that `hasGeometryPath()` returning `true` means there is a property named `"GeometryPath"`.